packages/react/package-lock.json

```diff
@@ -8962,6 +8970,18 @@
       "license": "MIT",
       "peer": true
     },
+    "node_modules/@ukic/web-components": {
```

Comment:
i'd be tempted to remove this, as it wasn't previously in the file. I would only push the changes for the added package